src/Site/Command/CreateSite.php

```diff
@@ -0,0 +1,41 @@
+<?php namespace Visiosoft\SiteModule\Site\Command;
+
+use Illuminate\Support\Str;
+use Visiosoft\SiteModule\Helpers\Formatters;
+use Visiosoft\SiteModule\Site\Contract\SiteInterface;
+use Visiosoft\SiteModule\Site\Contract\SiteRepositoryInterface;
+
+class CreateSite
+{
+    protected string $username;
+    protected int $server_id;
+    protected $basepath;
+    protected $php;
+
+    public function __construct(string $username, int $server_id, $basepath = null, $php = null)
+    {
+        $this->username = $username;
+        $this->server_id = $server_id;
+        $this->basepath = $basepath;
+        $this->php = $php;
+    }
+
+    public function handle(SiteRepositoryInterface $sites): SiteInterface
```

Comment:
It's not a console command or job. Why are you using handle practice?  I think if you use the basic class practice. It will be much easier to use. In this practice, you will have to trigger the function with dispatch or smth, which has nothing to do with queue.

Reply:
Actually, my goal was to create a queueable class. But I haven't added it yet. I edited and sent it. ✋